src/scripts.js

```diff
@@ -118,44 +107,128 @@ function displayTravelerData(data) {
 
 // Filter trips  -------------------------------------------------------------------------------------------
 
-
-function displayAllTravelerTrips(data) {
-  const allTravelerTrips = data.currentTraveler.travelerTrips;
-  domUpdates.resetAllTripsInnerHTML();
-  domUpdates.hideDisplayNotFoundMessage();
-  allTravelerTrips.forEach(trip => {
-    domUpdates.updateAllTravelerTrips(trip)
-  });
-};
-
-function displayUpcomingTravelerTrips(data) {
-  const upcomingTravelerTrips = data.currentTraveler.upcomingTrips;
-  if (!upcomingTravelerTrips.length) {
+function displayAllTravelerTrips(data, selector, arr) {
+  domUpdates.hideNotFoundMessage();
+  if (!arr.length) {
     domUpdates.displayNotFoundMessage()
   } else {
-    domUpdates.resetUpcomingTripsInnerHTML();
-    domUpdates.hideDisplayNotFoundMessage();
-    upcomingTravelerTrips.forEach(trip => {
-      domUpdates.updateUpcomingTravelerTrips(trip)
+    selector.innerHTML = ""
+    arr.forEach(trip => {
+      domUpdates.updateTravelerTrips(trip, selector)
     });
   }
 };
 
-function displayPendingTravelerTrips(data) {
-  const pendingTravelerTrips = data.currentTraveler.pendingTrips;
-  if (!pendingTravelerTrips.length) {
-    domUpdates.displayNotFoundMessage()
+// Form Page  -------------------------------------------------------------------------------------------
+
+function displayAndHideTripForm() {
+  domUpdates.displayAndHideFormPage()
+}; 
+
+function createDestinationList(data) {
+  destinationDropDown.innerHTML = "";
+  data.destinations.forEach(destination => {
+    domUpdates.displayDestinationDropDown(destination)
+  })
+}
+
+function displayTripQuote(e) {
+  e.preventDefault()
+  if (formDepartureDate.value < new Date().toISOString().split('T')[0]) {
+    domUpdates.displayDateError()
+  } else if (formNumTravelers.value <= 0 || !formNumTravelers.value) {
+    domUpdates.displayFormError()
+  } else if (formTripDuration.value <= 0 || !formTripDuration.value) {
+    domUpdates.displayFormError()
```

Comment:
These lines could possibly be combined if they're showing the same form error message.